### PR TITLE
Adding ccloud option to Quickstart.

### PR DESCRIPTION
**Pull request by ianjevans**
Closes DOC-3420.

### CODE REVIEW

File: cockroachcloud/quickstart.md [hunk 1]
```diff
@@ -30,31 +29,66 @@ Follow these steps to get started with CockroachDB using a {{ site.data.products
 
 </section>
 
-The sample code used in this tutorial is located in the [`quickstart-code-samples` GitHub repo](https://github.com/cockroachdb/quickstart-code-samples). This repo contains code samples written in JavaScript, Python, Go, and Java.
+<section class="filter-content" markdown="1" data-scope="serverless">
+
+<h2>Choose your installation method</h2>
+
+You can install a {{ site.data.products.serverless }} cluster using either the CockroachDB Cloud Console, a web-based graphical user interface (GUI) tool, or <code>ccloud</code>, a command line interface (CLI) tool.
```
Comment (stbof) [inline cockroachcloud/quickstart.md:36]:
```suggestion
You can install a {{ site.data.products.serverless }} cluster using either the CockroachDB Cloud Console, a web-based graphical user interface (GUI) tool, or <code>ccloud</code>, a command-line interface (CLI) tool.
```

File: cockroachcloud/quickstart.md [hunk 2]
```diff
@@ -30,31 +29,66 @@ Follow these steps to get started with CockroachDB using a {{ site.data.products
...
+
+<h2>Install <code>ccloud</code></h2>
+
+{% include cockroachcloud/ccloud/install-ccloud.md %}
+
+## Run `ccloud quickstart` to create a new cluster, create a SQL user, and retrive the connection string.
+
+{% include cockroachcloud/ccloud/quickstart.md %}
+
+Select the **General connection string**, then copy the connection string displayed and save it in a secure location. The connection string is the line starting `postgresql://`.
```
Comment (stbof) [inline cockroachcloud/quickstart.md:75]:
```suggestion
Select **General connection string**, then copy the connection string displayed and save it in a secure location. The connection string is the line starting `postgresql://`.
```

**Review by marksoper — CHANGES_REQUESTED** [review]
Need to apply the "-xz instead of -xJ" change here as well.

**Comment by ianjevans** [comment]
> Need to apply the "-xz instead of -xJ" change here as well.

Done in this commit: https://github.com/cockroachdb/docs/pull/13881/commits/b40715848b163c561e0f49f055bc9387891895c3

**Review by jenngeorge — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/13881)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @ianjevans, @marksoper, and @stbof)

___
*[`_includes/cockroachcloud/ccloud/quickstart.md` line 8 at r2](https://reviewable.io/reviews/cockroachdb/docs/13881#-N2YTeJZ3eQ3hEEcBaDe:-N2YTeJZ3eQ3hEEcBaDf:b-fwb49f) ([raw file](https://github.com/cockroachdb/docs/blob/6e18e1406a4abeb1bc24c0feb231769b056134e0/_includes/cockroachcloud/ccloud/quickstart.md#L8)):*
> ```Markdown
> ~~~
> 
> The `ccloud quickstart` command will open a browser window to log you in to CockroachDB Cloud. If you are new to CockroachDB Cloud, you can log in using your GitHub account, or create a new account using an email address.
> ```

I'm not sure if it will be clear that `ccloud quickstart` supports existing users logging in, but not account creation / new users signing up (yet).

What do you think about extending the sentence like "If you are new to CockroachDB Cloud, you should log in using your GitHub account, or create a new account using an email address before using the `ccloud quickstart` command."? 

Alternatively, how about adding a sentence like "If you are new to CockroachDB Cloud, first sign up using your GitHub account or an email address at https://cockroachlabs.cloud/signup (`ccloud` support for account creation is coming soon!)." before "Run `ccloud quickstart` and follow the instructions:"?

**Comment by ianjevans** [comment]
> What do you think about extending the sentence like "If you are new to CockroachDB Cloud, you should log in using your GitHub account, or create a new account using an email address before using the `ccloud quickstart` command."?
> 
> Alternatively, how about adding a sentence like "If you are new to CockroachDB Cloud, first sign up using your GitHub account or an email address at https://cockroachlabs.cloud/signup (`ccloud` support for account creation is coming soon!)." before "Run `ccloud quickstart` and follow the instructions:"?

I clarified, and removed the reference to GitHub as we now support multiple SSO options.

**Review by jenngeorge — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/13881)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @ianjevans, @marksoper, and @stbof)

___
*[`_includes/cockroachcloud/ccloud/quickstart.md` line 8 at r2](https://reviewable.io/reviews/cockroachdb/docs/13881#-N2YTeJZ3eQ3hEEcBaDe:-N2nEyTP4NaWVNcLxT84:bzbw68v) ([raw file](https://github.com/cockroachdb/docs/blob/6e18e1406a4abeb1bc24c0feb231769b056134e0/_includes/cockroachcloud/ccloud/quickstart.md#L8)):*
<details><summary><i>Previously, jenngeorge (Jennifer Georgevich) wrote…</i></summary><blockquote>

I'm not sure if it will be clear that `ccloud quickstart` supports existing users logging in, but not account creation / new users signing up (yet).

What do you think about extending the sentence like "If you are new to CockroachDB Cloud, you should log in using your GitHub account, or create a new account using an email address before using the `ccloud quickstart` command."? 

Alternatively, how about adding a sentence like "If you are new to CockroachDB Cloud, first sign up using your GitHub account or an email address at https://cockroachlabs.cloud/signup (`ccloud` support for account creation is coming soon!)." before "Run `ccloud quickstart` and follow the instructions:"?
</blockquote></details>

Thanks for updating "log in using your GitHub account" to "register using one of the single-sign-on options" for `cockroachcloud/ccloud-get-started.md` in commit https://github.com/cockroachdb/docs/pull/13881/commits/38808167a1754471eb53717b76b70276a5f81767 ! Do we want to make the same update in `_includes/cockroachcloud/ccloud/quickstart.md`?
___
*[`cockroachcloud/quickstart.md` line 71 at r3](https://reviewable.io/reviews/cockroachdb/docs/13881#-N2nCJY1BRWRcfBhUyAJ:-N2nCJY24BR9zJC078li:b-jlnfyr) ([raw file](https://github.com/cockroachdb/docs/blob/38808167a1754471eb53717b76b70276a5f81767/cockroachcloud/quickstart.md#L71)):*
> ```Markdown
> {% include cockroachcloud/ccloud/install-ccloud.md %}
> 
> ## Run `ccloud quickstart` to create a new cluster, create a SQL user, and retrive the connection string.
> ```

I think "retrive" should be "retrieve".

**Comment by ianjevans** [comment]
> Thanks for updating "log in using your GitHub account" to "register using one of the single-sign-on options" for `cockroachcloud/ccloud-get-started.md` in commit [3880816](https://github.com/cockroachdb/docs/commit/38808167a1754471eb53717b76b70276a5f81767) ! Do we want to make the same update in `_includes/cockroachcloud/ccloud/quickstart.md`?

Yes, good catch.

> I think "retrive" should be "retrieve".

Yup. Fixed. I wonder why our linter didn't catch that misspelling?

**Review by jenngeorge — APPROVED** [review]
:lgtm: thanks! 

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/13881#-:-N2nO2po1oc0nHtukVKz:b-1tfu21)* status: :shipit: complete! 1 of 0 LGTMs obtained (waiting on @ianjevans, @marksoper, and @stbof)

File: cockroachcloud/quickstart.md [hunk 3]
```diff
@@ -30,31 +29,66 @@ Follow these steps to get started with CockroachDB using a {{ site.data.products
...
+## Connect to the cluster
+
+</section>
+
+<section class="filter-content" markdown="1" data-scope="ccloud">
+
+Follow these steps to create a {{ site.data.products.serverless }} cluster using the <code>ccloud</code> CLI tool.
+
+{{site.data.alerts.callout_info}}
+The <code>ccloud</code> CLI tool is currently in beta.
```
Comment (stbof) [inline cockroachcloud/quickstart.md:64]:
```suggestion
The <code>ccloud</code> CLI tool is in beta.
```

**Pull request merged**